src/Diagnostics.RuntimeHost/Startup.cs

```diff
@@ -50,35 +52,49 @@ public void ConfigureServices(IServiceCollection services)
             var config = configManager.GetConfigurationAsync().Result;
             var issuer = config.Issuer;
             var signingKeys = config.SigningKeys;
-            services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme).AddJwtBearer(options =>
+            // Adding both custom cert auth handler and Azure AAD JWT token handler to support multiple forms of auth.
+            if (Environment.IsProduction())
             {
-                options.TokenValidationParameters = new TokenValidationParameters
-                {
-                    ValidateAudience = true,
-                    ValidAudience = Configuration["SecuritySettings:ClientId"],
-                    ValidateIssuer = true,
-                    ValidIssuers = new[] { issuer, $"{issuer}/v2.0" },
-                    ValidateLifetime = true,
-                    RequireSignedTokens = true,
-                    IssuerSigningKeys = signingKeys
-                };
+                services.AddAuthentication().AddCertificateAuth(CertificateAuthDefaults.AuthenticationScheme, 
+                    options =>
+                    {
+                        options.AllowedIssuers = Configuration["SecuritySettings:AllowedCertIssuers"].Split(",").Select(p => p.Trim()).ToList();
```

Comment:
Issuers can have commas in their name such as `CN=Hawk Co, OU=Hawk IT,L=Boston,S=Massachusetts,C=US`, split with pipes `|`